decidim-core/config/locales/en.yml

```diff
@@ -30,6 +30,7 @@ en:
         password_confirmation: Confirm your password
         personal_url: Personal URL
         remove_avatar: Remove avatar
+        tos_agreement: Tos agreement
```

Comment:
```suggestion
        tos_agreement: Terms and conditions of use agreement
```

Reply:
Thank you for your review @andreslucena, I updated the translation key and the related test! 
Please tell me if you have any additional remark 😁